File: two_sum.js
function twoSum(numbers, target) {
    //in coding it is a much better idea to keep things simple
    //we create the arr to store the pair
    let majestic=[]
    console.log(numbers)
    for(let i=0;i<numbers.length-1;i++){
      for(let j=i+1;j<numbers.length;j++){
        if(numbers[i]+numbers[j]===target){
          majestic.push(i)
          majestic.push(j)
          console.log(majestic)
          if(majestic.length==2){
             return majestic
            break
          }
        }
      }
    }
  }

File: two_sum.py
def two_sum(numbers, target):
    mysterio = []
    for i in range(len(numbers)-1):
        for j in range(i+1,len(numbers)):
            if numbers[i] + numbers[j] == target:
                mysterio.append(i)
                mysterio.append(j)
                return mysterio
                break
    
    
    
    # be careful about indentation rules in python